src/app/screens/LNURLPay.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState, MouseEvent } from "react";
+import axios from "axios";
+import sha256 from "crypto-js/sha256";
+import Hex from "crypto-js/enc-hex";
+import { parsePaymentRequest } from "invoices";
+
+import msg from "../../common/lib/msg";
+
+import Button from "../components/button";
+import PublisherCard from "../components/PublisherCard";
+
+type Props = {
+  details: {
+    minSendable: number;
+    maxSendable: number;
+    callback: string;
+  };
+  origin: {
+    name: string;
+    icon: string;
+  };
+};
+
+function LNURLPay({ details, origin }: Props) {
+  const [value, setValue] = useState<string | number>(details.minSendable);
+
+  async function confirm() {
+    try {
+      // Get the invoice
+      const params = {
+        amount: value, // user specified sum in MilliSatoshi
```

Comment:
we have to be careful here currently we use satoshis but we get MilliSatoshi here.